CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to hypersdk
+
+Thank you for your interest in contributing to `hypersdk`! By contributing to hypersdk, you are helping to build the foundation for the next generation of blockchains and decentralized applications.
+
+## Getting Started
+
+### Prerequisites
+
+To contribute to `hypersdk`, you'll need:
+
+- [Go](https://golang.org/dl/) 1.19 or higher
+
+### Setting up your development environment
+
+1. Clone the repository:
+
+```bash
+git clone https://github.com/ava-labs/hypersdk.git
+cd hypersdk
+```
+
+2. Install the dependencies:
+
+```go
+go mod download
+```
+
+This will download and install all required dependencies for the project.
+
+3. Build the project:
```

Comment:
I don't think there is actually any artifact that is built here (as it is just an SDK).

Reply:
You are right! I just added It based on a template that I had. Thank you for the feedback!